### ISSUE TITLE
Set a default credentials_manager

### ISSUE THREAD

**Issue by matthewdva** [openSUSE/osc#738]
Is there a method of setting a default credentials manager?  The goal would prevent the select_credentials manager_descr from prompting for specific back end if one is defined in the config file (and is a available).

If a method exists, the request would be document the procedure for the refactored credentials manager.

If a method does not exist, the request is to add this feature.

I have noticed that a credentials_mrg_class is added to the system profile section of the configuration file.  I did try placing the setting the general section of the configuration file without success.

For example, when using SecretStorage as the keyring backend the following is added:
[https://api.opensuse.org]
credentials_mgr_class = osc.credentials.KeyringCredentialsManager:keyring.backends.SecretService.Keyring

Placing the same line the general section has no effect.

**Comment by lethliel** [comment]
Yes. The credential_mgr_class is set per API. This has two reasons:

1. Each API has an own user/password combination.
2. We wanted to give the user the freedom to configure the backend per API.

So this would basically be a feature request.

**Comment by Thaodan** [comment]
I hit this issue today. I want to set the credentials manager API for all the API url I'm using.
My goal is to use keyring_pass for all of them.
There isn't a way to set a default as far as I know.
